pkg/data/dashboard/repo.go

```diff
@@ -38,19 +38,17 @@ func addClusterRepo(wrangler *wrangler.Context, repoName, branchName string) err
 	return err
 }
 
-// addClusterRepos sets up default cluster helm repositories in Rancher.
-// It attempts to add or update the cluster repositories 'rancher-charts', 'rancher-partner-charts',
-// and, if the RKE2 feature is enabled, 'rancher-rke2-charts'.
-func addClusterRepos(ctx context.Context, wrangler *wrangler.Context) error {
-	if err := addClusterRepo(wrangler, "rancher-charts", settings.ChartDefaultBranch.Get()); err != nil {
+// addRepos upserts the rancher-charts, rancher-partner-charts and rancher-rke2-charts ClusterRepos
+func addRepos(ctx context.Context, wrangler *wrangler.Context) error {
+	if err := addRepo(wrangler, "rancher-charts", settings.ChartDefaultBranch.Get()); err != nil {
 		return err
 	}
-	if err := addClusterRepo(wrangler, "rancher-partner-charts", settings.PartnerChartDefaultBranch.Get()); err != nil {
+	if err := addRepo(wrangler, "rancher-partner-charts", settings.PartnerChartDefaultBranch.Get()); err != nil {
```

Comment:
Changing this addRepo -> addClusterRepo might generate some problems, I remember someone from QA that tested this file.